### DISCUSSION TITLE
How to i18n emails in case of invalid recovery/verification?

### DISCUSSION THREAD

**Discussion by hugotiburtino** [ory/kratos#2681] [Q&A]
As in the [documentation](https://www.ory.sh/docs/kratos/emails-sms/custom-email-templates#nested-templates-i18n-customization) the suggested way to internationalize emails is using traits, as a result from this https://github.com/ory/kratos/issues/834. 

That works ok for valid emails, but what about the invalid emails (when v.g. a user wants to recovery account, but puts a wrong email)? In such cases, Kratos cannot (of course!) fetch the identity. Does anyone has a idea here how to solve that cases?

If we had the choice, we would do it quite simple by getting the current domain in which the user is (`en.serlo.org` for English or `es.serlo.org` for Spanish) and sending the correct email then.

**Comment by vinckr** [comment]
Hey Hugo, 
apologies for the late reply! 
Did you solve this issue in the meantime? At the moment I am not sure what the best way to do this is. 

In the case of `invalid` emails the user is not signed up, can you be confident that the domain is the correct language for them?
